react/RuiExpandCollapseReact/src/RuiExpandCollapse.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Deloitte Digital 2019
+ *
+ * This source code is licensed under the BSD-3-Clause license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { createRef } from 'react';
+import '@rhythm-ui/expand-collapse';
+
+import {
+	IRuiExpandCollapseProps,
+	IHTMLRuiExpandCollapseElement,
+	IExpandCollapseDetailsProps,
+	IExpandCollapseSummaryProps,
+} from './IRuiExpandCollapse';
+
+export class RuiExpandCollapse extends React.Component<IRuiExpandCollapseProps> {
+	private ruiExpandCollapseEl = createRef<IHTMLRuiExpandCollapseElement>();
+
+	public static Details = class extends React.Component<IExpandCollapseDetailsProps> {
```

Comment:
Can we please add simple jsdoc above each of these methods. Because we are not a chop shop :)

Reply:
Also looks like we can make this a functional component

```
static Details: React.FC<IExpandCollapseDetailsProps> = () => {
      ...
}
```

Reply:
Actually on this topic. I think this file will be more human readable if we seperate these helper components.

Suggest:

```
export const Details = () => {}

export class ExpandCollapsy extends Component {

         public static Details = Details;

}

```

cc @jeffdowdle

Reply:
@jennasalau  updated